index.js

```diff
@@ -261,6 +261,7 @@ low.registerLanguage(
 )
 low.registerLanguage('sml', require('highlight.js/lib/languages/sml'))
 low.registerLanguage('sqf', require('highlight.js/lib/languages/sqf'))
+low.registerLanguage('sql_more', require('highlight.js/lib/languages/sql_more'))
```

Comment:
Not sure this is sticking around, but we have it for now. :)  Gonna be stuck until v11.